<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialAssignDialog</class>
 <widget class="QDialog" name="MaterialAssignDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Assign Material</string>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: #303645;
    color: #a4b0bd;
    font-weight: 600;
    padding-left: 8px;
}</string>
     </property>
     <property name="text">
      <string>Assign Material</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bodyLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="materialRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="materialLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Material:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="materialCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertPolicy::NoInsert</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="colorRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="baseColorLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Base Color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorSwatchButton">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Current base color</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pickColorButton">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Pick</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="colorRowSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="hintLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: #6b7580; padding-left: 2px; }</string>
       </property>
       <property name="text">
        <string>If no selection, Assign applies to all polygons.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <spacer name="buttonSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="assignButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Assign</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
